packages/webapp/components/layouts/ProfileLayout/index.tsx

```diff
@@ -57,6 +57,7 @@ export default function ProfileLayout({
     process.env.NEXT_PUBLIC_API_URL,
   );
   ogImageUrl.searchParams.set('type', 'wide');
```

Comment:
Not part of this PR but wanted to ask, are we only going to use the wide version for OG?

Reply:
As far as I understand, yes: https://www.figma.com/file/nmfWPS7x3kzLUvYMkBx2kW/daily.dev---Dev-Mode?node-id=4234%3A17279&mode=dev